### ISSUE TITLE
Dead link to "SpaceWar" in Samples & Demos documentation

### ISSUE THREAD

**Issue by davidhesselbom** [MonoGame/MonoGame#7662]
MonoGame/Documentation/articles/samples.md has a link to https://github.com/MonoGame/MonoGame.Samples/tree/develop/SpaceWar, which does not exist.

https://github.com/MonoGame/MonoGame/blob/4f2a554fe890000e9a2456c776f658824a3c4858/Documentation/articles/samples.md?plain=1#L11

#### What version of MonoGame does the bug occur on:
- MonoGame 3.7

#### What operating system are you using:
- Windows

#### What MonoGame platform are you using:

- DesktopGL

**Comment by NogginBox** [comment]
Has the sample been moved, or discontinued? I've had a quick bingle, but can't find it.

**Comment by KakCAT** [comment]
It seems it was removed here, but I've found no explanation why.

https://github.com/MonoGame/MonoGame.Samples/commit/7524b32174e4ccf42abe47eb58d294c76871d393

**Comment by mrhelmut** [comment]
I can't remember why the SpaceWar sample has been removed, but I assume that it was because the networking features of MonoGame had been removed and this was the networking sample. @SimonDarksideJ likely knows.

We'll be updating the documentation for 3.8.1 and upgrading the samples.

**Comment by SimonDarksideJ** [comment]
SpaceWar was mainly removed as it was not fully functional, but I believe the intent was to put it on a development branch or replace it with a better sample.
But I'll be sure to remove the dead link in the docs, that shouldn't be there.
#7693

**Comment by SimonDarksideJ** [comment]
For prosperity, I resurrected the old SpaceWar sample in to it's own branch.  It's not fully finished and still needs work, but at least devs can still access it.  I'll find some time to work with @CartBlanche who I believe has a more up to date version of the shaders which solved some of the issues in the sample.
https://github.com/MonoGame/MonoGame.Samples/tree/spacewar

**Comment by CartBlanche** [comment]
@SimonDarksideJ I looked through my Git Repo and don't believe I have an updated Spacewar. I did have a slightly better working version of that 3D like Descent XNA demo, that we worked on a little. But that needs further tweaks to get fully working, same with the 3D car racing demo. Texture and possibly shader issues.

If networking was the issue, do we have a "built-in" solution or suggested sample we can hook up for people new to networking?

**Comment by SimonDarksideJ** [comment]
Ahh yes, it was the Racing Game you were working on, not SpaceWar, my bad.

My main concern is that we don't have a 3D sample in the repo at the moment.  Although I did upgrade one of the XNA samples for another project, so I'll see if I can dig that out.

A networking sample would be awesome but since GamerServices was removed I don't think we have a good MonoGame sample for that.  Could reach out to @NemoKradXNA so see if he can contribute something from his recent efforts with PlayFab
